gptel-transient.el

```diff
@@ -252,14 +269,10 @@ which see."
           (goto-char (point-min))
           (forward-line 1)
           (while (not (eobp))
-            (when-let ((act (read (current-buffer))))
-              (forward-char)
-              (save-excursion
-                (while (re-search-forward "\"\"" (line-end-position) t)
-                  (replace-match "\\\\\"")))
-              (when-let ((prompt (read (current-buffer))))
-                (puthash act prompt gptel--crowdsourced-prompts)))
-            (forward-line 1)))))
+	    (when-let ((act (gptel--read-csv-column)))
```

Comment:
Use a single `when-let*` or `and-let*` instead of two `when-let` clauses.  (Also, `when-let` is deprecated in favor of `when-let*` in upcoming Emacs versions)